sections/futures/MarketDetails/MarketDetails.tsx

```diff
@@ -66,6 +66,14 @@ const MarketDetails: React.FC<MarketDetailsProps> = ({ baseCurrencyKey }) => {
 	const pastPrice = dailyPriceChanges.find((price: Price) => price.synth === baseCurrencyKey);
 
 	const data: MarketData = React.useMemo(() => {
+		const fundingTitle = `${
+			fundingRateQuery.failureCount > 0 && !avgFundingRate && !!marketSummary ? 'Inst.' : '24H'
```

Comment:
We might have to come back to this for i18n purposes (seems like a pain here though lol)

Reply:
agreed, I think we could just pull translations here instead but we're also talking about changing to 1 hr. Resolving this for now, can revisit when we make those changes.